Affiliates.html

```diff
@@ -0,0 +1,65 @@
+﻿<!doctype html>
+<html>
+<head>
+    <title>Affiliates</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <!-- Normalize CSS -->
+    <link rel="stylesheet" href="CSS/Normalize.css" />
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+    <link rel="stylesheet" href="CSS/Site.css">
+    <link rel="stylesheet" href="CSS/public_documents.css">

```

Comment:
This css was specifically made for the public_documents.html file.

I think the tbody > tr > td ul CSS styling can be moved into the site.css and applied to all tables rather than just specific tables.

So you may remove this line and I will make the changes to the css.